wemake_python_styleguide/constants.py

```diff
@@ -379,3 +379,15 @@
 
 # Used to specify as a placeholder for `__init__`:
 INIT: Final = '__init__'
+
+# Used to specify unreadable combinations of letters and letters with numbers
+UNREADABLE_CHARACTER_COMBINATIONS = frozenset((
```

Comment:
It should be annotated with `: Final` and documented in the public section with `#:`